components/rmrk/Gallery/Item/ActionList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="buttons">
+    <b-button
+      v-for="action in actions"
+      :key="action"
+      :type="iconType(action)"
+      outlined
+      expanded
+      @click="handleActionSelect(action)">
+      {{ action }}
+    </b-button>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue } from 'nuxt-property-decorator'
+import {
+  getActionButtonColor, ShoppingActions
+} from '@/utils/shoppingActions'
+
+@Component({})
+export default class ActionList extends Vue {
+  @Prop({ type: Array, required: false }) public actions!: ShoppingActions[]
+
+  @Emit('click')
+  protected handleActionSelect(action: ShoppingActions) {
+    return action
+  }
```

Comment:
`ShoppingActions` is an array? or singleton

Reply:
cc @Jarsen136 ;D 
a092700295e1da509c2059be3fc9a213e216dc6a

Reply:
yup, It's my commit : ) 
removing the `s` would be better. 
Maybe let's keep it so and I will remove it in the future